### DISCUSSION TITLE
How to measure facet areas and sides length on a decimation

### DISCUSSION THREAD

**Discussion by johnsculp** [cnr-isti-vclab/meshlab#1017] [Q&A]
Hello 

I make sculpture and came across meshlab and this discussion site, I know little of 3D computing or this meshlab app.
See attached photos of faceted items. I make sculptures sometimes very large ones in sheet metal in a facet style. Normal every facet sheet is measured to fit a space and then cut by manual guillotine. I wanted to know if Meshlab could tell me the sizes of the facets on this decimated boot mesh, the area of each facet, length of each side, angles? Could the mesh be flattened or opened out like a large skin and each facet exported as a dxf file for cutting out by machine? Or is this capability just a dream?

regards John

![Boot render faceted](https://user-images.githubusercontent.com/85869343/121863492-d1911e80-ccf3-11eb-9884-e95bc38ffc51.png)

![B591EADB-E5CA-41F0-8323-ADAB8CE05E42_1_201_a](https://user-images.githubusercontent.com/85869343/121863711-0b622500-ccf4-11eb-952b-14423e938b58.jpeg)

**Comment by alemuntoni** [comment]
Wow, very beautiful sculptures!! :)
About the infos you need, you could use the info tool of meshlab, and after selecting a face, press P. Some more information about the selected face will be printed. However, info like edge length, angles and face area are still missing. However they could be easily added in future versions of meshlab!
For the flattining, we still don't have this kind of filter, sorry!

**Reply by johnsculp** [reply]
Thankyou for that. I will give it a try.
